server/eclipse-project/src/main/java/us/freeandfair/corla/hibernate/Persistence.java

```diff
@@ -40,6 +41,7 @@
 import us.freeandfair.corla.model.Contest;
 import us.freeandfair.corla.model.Elector;
 import us.freeandfair.corla.model.UploadedFile;
+import us.freeandfair.corla.util.Pair;
 
 /**
  * Manages persistence through Hibernate, and provides several utility methods.
```

Comment:
I feel like I need more of an explanation for the legitimate use of the helper methods in this class (someday).